src/Containers/App/style.css

```diff
@@ -33,6 +33,8 @@ body {
   font-family: sans-serif;
   font-weight: 700;
   line-height: initial;
+  margin-top: 50vh;
```

Comment:
Again fixing styling when entering the homepage from and external link or by typing in the address bar. Translation is to accommodate centering in the absence of
`style={collapsed ? {} : { width: parentWidth, height: parentHeight }}`